src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TIME_DISC = 50;
+  const MIDL_TIME_DISC = 20;
+  const PRICE = 40;
+  const LONG_TIME = 7;
+  const MIDL_TIME = 3;
+
+  if (days >= LONG_TIME) {
+    return PRICE * days - LONG_TIME_DISC;
+  } else if (days >= MIDL_TIME) {
+    return PRICE * days - MIDL_TIME_DISC;
```

Comment:
The `else if` block is unnecessary because the previous `if` block contains a `return` statement. You can simplify this by removing the `else` keyword.